docs/api/meetings.md

```diff
@@ -0,0 +1,636 @@
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 79
+                },
+                {
+                    "name":"Jessica Smith",
+                    "email": "jessicas@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 133
+                },                    
+                {
+                    "name": "Chris Hawthorne",
+                    "email": "chrish@dahliacorp.com",
+                    "external": true,
+                    "status": "accepted"
+                },
+                {
+                    "name": "Ada Bottas",
+                    "email": "adab@dahliacorp.com",
+                    "external": true,
+                    "status": "pending"
+                },
+                {
+                    "name": "Conrad Lee",
+                    "email": "conradl@agenda.com",
+                    "external": false,
+                    "status": "declined",
+                    "id": 95
+                }
+            ],
+            "name": "Sync up",
+            "description": "A sync up with the Dahlia Corporation to discuss the Agenda implementation.",
+            "createdAt": 1667302643,
+            "updatedAt": 1667302643,
+            "createdBy": 35,
+            "meetingTime": 1667489400,
+            "series": false
+        }
+    ],
+    "page": 1
+}
+```
+
+| Property | Description |
+| -------- | ----------- |
+| `id` | The ID of the meeting. |
+| `source` | An object that contains details about the meeting's location. |
+| `source.name` | The name of the source. |
+| `source.category` | The category of the source. This value can either be `virtual` for online sources or `in-person` for in-person sources (such as meeting rooms). |
+| `source.type` | The type of the source. Possible values include `zoom`, `teams`, `slack`, and `skype`. This value is **only** returned for sources in the `virtual` category. 
+| `source.url` | The URL for the source. This value is **only** returned for sources in the `virtual` category.
+| `source.id` | The ID of the source. |
+| `invitees` | An array of people invited to the meeting. |
+| `invitees.name` | The name of the person invited. |
+| `invitees.email` | The email of the person invited. |
+| `invitees.external` | A boolean value that shows whether or not the person belongs to the company. |
+| `invitees.status` | The status of the person invited to the meeting. This value can be `accepted`, `pending`, or `declined`. |
+| `invitees.team` | If applicable, the ID of the team the invitee belongs to. This value is only returned for non-external invitees and if the entire team is invited to the meeting. |
+| `invitees.id` | The ID of the person invited. This value is only returned for non-external invitees. |
+| `name` | The name of the meeting. |
```

Comment:
I think `title` or `topic` would be better names for this since that's what existing clients call this field in their UIs